pkg/actors/actors.go

```diff
@@ -992,11 +1083,17 @@ func (a *actorsRuntime) executeReminder(reminder *Reminder) error {
 	log.Debugf("executing reminder %s for actor type %s with id %s", reminder.Name, reminder.ActorType, reminder.ActorID)
 	req := invokev1.NewInvokeMethodRequest(fmt.Sprintf("remind/%s", reminder.Name))
 	req.WithActor(reminder.ActorType, reminder.ActorID)
-	req.WithRawData(b, invokev1.JSONContentType)
 
 	policy := a.resiliency.ActorPreLockPolicy(context.Background(), reminder.ActorType, reminder.ActorID)
 	return policy(func(ctx context.Context) error {
-		_, err := a.callLocalActor(ctx, req)
+		var err error
+		if a.isInternalActor(reminder.ActorType) {
```

Comment:
I think it is better to make each ActorType to declare its contentType, so this logic is generic and not specific for InternalActors.

Reply:
I think the right way to introduce something like this would be as a new property in `EntityConfig`. But again, I worry that this is out of scope for this PR. Rather, I think this needs to be a separate feature that's created and documented independently, and that we should just continue hardcoding the content types as we have already been doing.